components/popover/test/popover-mixin.test.js

```diff
@@ -0,0 +1,30 @@
+import './popover.js';
+import { fixture, oneEvent, runConstructor } from '@brightspace-ui/testing';
+
+describe('popover-mixin', () => {
+
+	describe('constructor', () => {
+
+		it('should construct', () => {
+			runConstructor('d2l-test-popover');
+		});
+
+	});
+
+	describe('events', () => {
+
+		it('should fire the open event when the opened attribute is set to true', async() => {
+			const elem = await fixture('<d2l-test-popover></d2l-test-popover>');
+			elem.opened = true;
+			await oneEvent(elem, 'd2l-popover-open');
+		});
+
+		it('should fire the close event when the opened attribute is set to true', async() => {
```

Comment:
```suggestion
		it('should fire the close event when the opened attribute is set to false', async() => {
```

Reply:
nit: Since it's also in the component logic, it could be good to have a test for if `<d2l-test-popover></d2l-test-popover>` has `opened` assigned to false (from undefined) then it does not fire the close event.

Reply:
I added event tests for both the opened and closed initial states.